script/emailData.js

```diff
@@ -0,0 +1,6 @@
+//사용중인 유저 데이터
+const userData = [
+  { email: "test@codeit.com", password: "codeit101" },
+  { email: "kymgok1001@naver.com", password: "dksdkffuwna1" },
+];
```

Comment:
일반적으로 상수값을 쓸 때는 일반 변수와 다르게 PASCAL_CASE 등을 사용해서 상수임을 명확하게 드러내는걸 권장하는데요. 참고 하시면 좋을거 같네용 ㅎㅎ
```suggestion
const USER_DATA = [
  { email: "test@codeit.com", password: "codeit101" },
  { email: "kymgok1001@naver.com", password: "dksdkffuwna1" },
];
```